src/index.mjs

```diff
@@ -19,34 +19,28 @@ class Game {
   }
 
   start() {
-    this.name = prompt("Enter your name:") || "Guest";
+    this.name = prompt('Enter your name:') || 'Guest';
     this.displayMenu();
   }
 
   handleMenuClick = function (event) {
     switch (event.target.dataset?.val) {
-      case "1":
+      case '1':
         this.updateLevel(1);
         this.gameLoop();
         break;
-      case "2":
-        console.log("Will Show Leaderboard Now...");
+      case '2':
+        console.log('Will Show Leaderboard Now...');
         break;
-      case "3":
-        this.name = prompt("Enter name to be updated:") || "Guest";
+      case '3':
+        this.name = prompt('Enter name to be updated:') || this.name;
         this.displayMenu();
     }
   }.bind(this);
 
   displayMenu() {
-    this.container.innerHTML = `Welcome ${this.name},
-    <ol>
-      <li data-val="1">Start New Game</li>
-      <li data-val="2">See Leaderboard</li>
-      <li data-val="3">Update Name</li>
-    </ol>`;
-    this.container.removeEventListener("click", this.handleMenuClick);
-    this.container.addEventListener("click", this.handleMenuClick);
+    document.querySelector('.username').innerText = `${this.name}`;
```

Comment:
This probably makes it non-generic. Does not allow multiple instances to be on the same, page?